applications/app/views/videoEmbed.scala.html

```diff
@@ -124,5 +124,10 @@
                 </script>
             }
         }
+
+        @{
+            page.model.map(videoPage => fragments.analytics.base(videoPage))
```

Comment:
Shouldn't `page` be passed to `base`? To make that possible, `EmbedPage` could inherit from either `StandalonePage` or `ContentPage`. I notice `EmbedPage` has an optional model, if we could tweak this to be non-optional, we could easily write `EmbedPage extends ContentPage`